flow360/component/simulation/outputs/outputs.py

```diff
@@ -210,7 +389,55 @@ class SurfaceIntegralOutput(Flow360BaseModel):
 
 
 class ProbeOutput(Flow360BaseModel):
-    """:class:`ProbeOutput` class for probe monitor output settings."""
+    """
+    :class:`ProbeOutput` class for setting output data probed at monitor points.
+
+    Example
+    -------
+
+    - Define :class:`ProbeOutput` on multiple monitor points.
+
+    >>> fl.ProbeOutput(
+    ...     name="probe_group_points",
+    ...     entities=[
+    ...         fl.Point(
+    ...             name="Point_1",
+    ...             location=(0.0, 1.5, 0.0) * fl.u.m,
+    ...         ),
+    ...         fl.Point(
+    ...             name="Point_2",
+    ...             location=(0.0, -1.5, 0.0) * fl.u.m,
+    ...         ),
+    ...     ],
+    ...     output_fields=["primitiveVars"],
+    ... )
+
+
+    - Define :class:`ProbeOutput` on monitor points along the line.
+      :code:`Line_1` is from (1,0,0) to (1.5,0,0) and has 6 monitor points.
+      :code:`Line_2` is from (-1,0,0) to (-1.5,0,0) and has 3 monitor points.
```

Comment:
Can you list the points explicitly here? You can change -1.5 to for example -1.9 so the coordinates have finite digits.

Reply:
Also add units here.